src/main/java/seedu/address/ui/HelpEditWindow.java

```diff
@@ -0,0 +1,106 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import javafx.stage.Stage;
+import seedu.address.commons.core.LogsCenter;
+
+/**
+ * Controller for a help page
```

Comment:
I think this javadocs needs a full stop at the end

Reply:
yes i agree!! i have added them